### PR TITLE
Add hack to compile rhsm.certificate on login

### PR DESCRIPTION
**Pull request by kahowell**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -223,3 +223,9 @@
   become: yes
   tags:
     - xauth
+
+- name: add rhsm.certificate compile hack to .bashrc
+  lineinfile:
+    path: "{{ ansible_env.HOME }}/.bashrc"
+    line: "test -f {{ subman_checkout_dir }}/src/rhsm/*certificate*.so || (cd {{ subman_checkout_dir}}; {{ subman_python_interpreter }} setup.py build_ext --inplace 2>&1 > /dev/null)"
```
Comment (jirihnidek) [inline tasks/main.yml]: When source code (`certificate.c`) is newer then target file `*certificate.so`, then it is also necessary to compile this file.

**Review by jirihnidek — CHANGES_REQUESTED** [review]
This could work, but could expect automatic rebuilding od certificate.so, when source code is changed.

**Comment by kahowell** [comment]
d71c555..accaf6b: change hack to use mtime.

@jirihnidek does this work for you?

**Review by jirihnidek — APPROVED** [review]
ACK

**Pull request merged**